tracing-core/src/metadata.rs

```diff
@@ -380,6 +381,11 @@ impl Kind {
     /// `Span` callsite
     pub const SPAN: Kind = Kind(KindInner::Span);
 
+    /// `enabled!` callsite. [`Collect`][`crate::collect::Collect`]s can assume
```

Comment:
docs needs to be updated
```suggestion
    /// `enabled!` callsite. [`Subscriber`][`crate::subscriber::Subscriber`]s can assume
```